test/versioned/fastify/package.json

```diff
@@ -8,10 +8,28 @@
         "node": ">=12"
       },
       "dependencies": {
-        "fastify": ">=2.0.0"
+        "fastify": ">=2.0.0 < 3",
+        "middie": "5.3.0"
       },
       "files": [
-        "integration.tap.js"
+        "errors.tap.js",
+        "fastify-2-naming.tap.js",
+        "new-state-tracking.tap.js"
+      ]
+    },
+    {
+      "engines": {
+        "node": ">=12"
+      },
+      "dependencies": {
+        "fastify": ">=3.0.0",
+        "middie": "5.3.0"
+      },
+      "files": [
+        "errors.tap.js",
+        "fastify-3.tap.js",
+        "fastify-3-naming.tap.js",
```

Comment:
Split into v2 and v3+ run configurations since I had a few types of tests now that forked. Cleaned up the in-test forks for the naming files that split.